MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -207,6 +207,29 @@ - (void)processRequest:(NSURLRequest *)request {
             }
 
             NSUserDefaults.mme_configuration.mme_configUpdateDate = MMEDate.date;
+
+            NSString *digest = nil;
+            NSString *digestHeader = httpResponse.allHeaderFields[@"Digest"];
+            if (digestHeader) {
+                // Look for 'SHA-256' hash in case of multiple different digest values
+                NSArray *digestValues = [digestHeader componentsSeparatedByString:@","];
+                if (digestValues.count >= 2) {
```

Comment:
NSArray is now `key=value`, right?
could we do something like (with some additional checks):
```
    NSArray *kvs = [digest componentsSeparatedByString:@","];
    NSMutableDictionary *dict = [NSMutableDictionary dictionary];
    for (NSString *field in kvs) {
        NSArray *keyValue = [field componentsSeparatedByString:@"="];
        dict[keyValue[0]] = keyValue[1];
    }
```
to get a cleaner structure like:
```
{
    Digest = abcd;
    "SHA-256" = 1234;
}
```
or are there too many variations in the data?

Reply:
Unfortunately, the current config service is not sending the `digest` header as `<key>=<value>` format, and that's the reason we need handle the different corner cases to fallback to the value(just hash) received from the config service.